index.d.ts

```diff
@@ -0,0 +1,13 @@
+import unified from 'unified'
+import unist from 'unist'
+import hast from 'hast'
+
+
+interface RemarkAutolinkHeadingsOptions {
+  behavior?: 'prepend' | 'append' | 'wrap' | 'before' | 'after'
+  linkProperties?: Record<string, any>
+  content?: any
```

Comment:
based off https://github.com/remarkjs/remark-autolink-headings#optionscontent this probably would be

```suggestion
  content?: Element | Element[] | () => Element | Element[]
```